src/Components/ServiceScene/Breakdowns/SelectLabelActionScene.tsx

```diff
@@ -73,6 +190,36 @@ export class SelectLabelActionScene extends SceneObjectBase<SelectLabelActionSce
             Select
           </Button>
         )}
+
+        {popover && (
+          <PopoverController content={<popover.Component model={popover} />}>
+            {(showPopper, hidePopper, popperProps) => {
```

Comment:
Popover?

Reply:
https://developers.grafana.com/ui/latest/index.html?path=/story/pickers-and-editors-colorpicker-popovers--basic
It's a grafana UI element, but I think I'm misunderstanding the question

Reply:
Unexpected! Thanks for the info.